### PR TITLE
Accept full URLs as input

### PR DESCRIPTION
**Pull request by realabbas**
Input Domain Url doesn't work when we input it as follows

``` www.domain.com/ ```  or  ``` www.domain.com/domain-new-page/```

image

**AFTER Changes**

Now it just takes the string ```www.domain.com``` and eliminates the rest.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const statusCodes = {
 };
 
 module.exports = async url => {
-	url = url.replace(/^(?:https?:)?\/\//, '');
+	url = url.substring(url.indexOf('www'), url.indexOf('.com') + 4);
```
Comment (sindresorhus) [inline index.js]: Would be better to use `new URL(url)` and then get the host.

**Comment by sindresorhus** [comment]
Can you add a test?

File: index.js (hunk 1 above)
Comment (realabbas) [reply index.js]: Yes, it would be better. Adding this change now

**Comment by realabbas** [comment]
> Can you add a test?

Hey, @sindresorhus I think the possible test cases are already written in **test.js**?

**Comment by realabbas** [comment]
@sindresorhus I don't understand why the tests are failing. Can you help me out ?

**Comment by realabbas** [comment]
Thank you for accepting the PR @sindresorhus 😉

**Pull request merged**